### PR TITLE
meters cleanup

### PR DESCRIPTION
**Pull request by lburgazzoli**
- metrics: remove histograms and percentiles
- metrics: remove unused tags

### CODE REVIEW

File: cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java [hunk 1]
```diff
@@ -305,7 +305,6 @@ private UpdateControl<ManagedConnector> measure(
             .tags(tags)
             .tag("cos.connector.id", connector.getSpec().getConnectorId())
             .tag("cos.deployment.id", connector.getSpec().getDeploymentId())
```
Comment (lgarciaaco) [inline cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java:307]: We probably can also remove the deployment_id, we dont use it, @shivamG640 to confirm

Comment (lburgazzoli) [reply cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java:307]: @lgarciaaco @shivamG640 do we actually use cos.connector.id and cos.deployment.id ?

Comment (lgarciaaco) [reply cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java:307]: deploymentId you can delete but @shivamG640 should confirm that dashboards dont use it. Connector id we need

Comment (lburgazzoli) [reply cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java:307]: For which reason we use it ?
Note that this is the operator, no the sync

Comment (shivamG640) [reply cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java:307]: Sorry for missing the old conversation!
@lgarciaaco, I could only find the usage of these metrics on this [file](https://github.com/bf2fc6cc711aee1a0c2a/cos-observability-resources/blob/main/resources/grafana/overview.yaml#L502)
@lburgazzoli, This metric is related to reconciliations and we don't monitor reconciliation per connectors so we can remove the connector_id and deployment_id.

**Comment by lburgazzoli** [comment]
@lgarciaaco @shivamG640 @valdar mind having a look ? I've removed all the connector related tags (connector-id, deployment-id, organization and pricing tier) from the fleetshard camel and debezium as it looks like we don't use such information anywhere.

**Review by valdar — APPROVED** [review]

**Review by lgarciaaco — APPROVED** [review]

**Review by shivamG640 — APPROVED** [review]

**Pull request merged**